.github/workflows/auto-commit-client-updates.yml

```diff
@@ -32,6 +32,7 @@ jobs:
         git checkout ${GITHUB_REF##*/}
         make docker_run
         echo -e "Make execution completed."
+        git add spec/oas3.patched/.
```

Comment:
Do we want to do this or #57? (or this now, 57 later)

Reply:
I think we should do this now, and maybe 57 later.  Doing this now makes it easer to open & review spec update PRs:  easier to open because all you have to do is update the `spec/oas3.fetched` directory, easier to review because the `spec/oas3.patched` directory will always be up-to-date, so it's easier to notice if the spec update had unintended consequences on the patching process.